erts/preloaded/src/erts_internal.erl

```diff
@@ -484,6 +485,49 @@ perf_counter_unit() ->
 is_system_process(_Pid) ->
     erlang:nif_error(undefined).
 
+-spec sys_proc_outstanding_requests_limit() -> pos_integer().
+
+sys_proc_outstanding_requests_limit() ->
```

Comment:
Why have this function and not use erlang:system_info?

Reply:
It is frequently used by the code purger and the literal collector, so we want it efficient for that use case. In order to implement it efficiently in `system_info()` we need to move it up in the `if` branching mess in `system_info()` which would make other things less efficient that the user actually might want to look at frequently. It could be interesting for the user to look at this setting, but hardly more than once when checking their configuration. I haven't looked that closely to what is at the top of `system_info()` though, it might perhaps be moved up near the top.

Reply:
I moved the argument up the `if` branching in `system_info` and removed the `erts_internal` function.